packages/procaptcha-pow/src/components/ProcaptchaWidget.tsx

```diff
@@ -41,6 +44,41 @@ const Procaptcha = (props: ProcaptchaProps) => {
 		}
 	}, [config.language]);
 
+	// Add event listener for the execute event (works for invisible mode)
+	useEffect(() => {
+		// Only set up event listener if in invisible mode
+		if (config.mode === "invisible") {
```

Comment:
enum / type

Reply:
since you're using Zod you can use `Mode.Values.invisible` or something like that